src/sections/Meshery/Features-Col/index.js

```diff
@@ -74,12 +76,13 @@ const Features = () => {
         </h1>
       </TitleContainer>
       <FeaturesSectionContainer>
-        {data.features.map((feature, index) =>
+        {LifecycleFeature().features.map((feature, index) =>
```

Comment:
Instead of using the function here directly, save the return value in a variable like `const data = LifecycelFeature().features` and then use the `data` varaible here.